mono/mini/mini-posix.c

```diff
@@ -1080,6 +1079,8 @@ dump_native_stacktrace (const char *signal, void *ctx)
 			exit (1);
 		}
 
+		mono_debugger_agent_send_crash (output, &hashes, 2 /* wait # seconds */);
```

Comment:
That is a big change which has nothing to do with the new version of the Crash Dumper. Split that off of this PR.